src/appengine/handlers/cron/cleanup.py

```diff
@@ -839,10 +893,10 @@ def update_fuzz_blocker_label(testcase, issue,
       'to be found.' % (platform_message, fuzzer_name))
   if utils.is_oss_fuzz():
     update_message += OSS_FUZZ_INCORRECT_COMMENT
```

Comment:
this needs update to use _append_generic_incorrect_comment ? Can OSS_FUZZ_INCORRECT_COMMENT be removed, if not, it is ok.

Reply:
oss-fuzz is a special case, because instead of a label we point to the github issues url. not sure how best to generalise this..

Reply:
Maybe add another config variable to customize that type of message?